src/libs/actions/IOU.js

```diff
@@ -348,6 +401,8 @@ function createSplitsAndOnyxData(participants, currentUserLogin, amount, comment
             oneOnOneChatReport.iouReportID = oneOnOneIOUReport.reportID;
         }
 
+        const oneOnOneTransaction = TransactionUtils.buildOptimisticTransaction(NumberUtils.roundDownToTwoDecimalPlaces(amountInCents / (participants.length + 1)), currency);
```

Comment:
We use [this logic](https://github.com/Expensify/App/blob/30420a19d14bbc021189d9bbfe7b702fcd1c2323/src/libs/actions/IOU.js#L314) for the splitAmount. How come we are using something different here?

Reply:
No reason but my own foolishness. Fixed now